examples/_logger/logrus/go.mod

```diff
@@ -16,11 +16,11 @@ require (
 	github.com/klauspost/compress v1.13.6 // indirect
```

Comment:
Ran the various logger examples to ensure that the driver's connection ID was setting properly. This is options, so there are no unified spec tests for this. There were multiple indirect go.mod updates required by the example modules.